spec/features/gym_search/index_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'gyms near me page' do
+  before do
+    visit dashboard_path(@user.id)
+
+    click_on 'Find Gyms Near Me'
+  end
+
+  it 'can find gyms near me', :vcr do
+    expect(page).to have_content("Gyms Near #{@user.zip_code}")
+
+    within '#gyms' do
+      expect(page).to have_css('div', count: 20)
+
+      within '#BJBXzKYxQAXZKb5W6HrRnA' do
+        expect(page).to have_content("Rishi's Community Yoga")
+        expect(page).to have_content('2750 S Wadsworth Blvd, Bldg D, Ste 202, Denver, CO 80227')
+        expect(page).to have_content('(303) 601-0789')
+      end
```

Comment:
For "future-proofing", I wonder if we could make this test more dynamic?  For example, what happens if the search results change because a gym is added or this gym goes out of business?

One thought is that the facade/service spec tests would test that the API calls are working as expected.  And then for the view/feature test, we can either stub the search results or we can call the API and test that the results are displayed on the view page in the order we want ... without needing to hardcode the search results.

Perhaps we just create an `issue` for this to address down the line.  Otherwise, seems to be working as expected!